### ISSUE TITLE
Wallabag app has no stylesheets or logos

### ISSUE THREAD

**Issue by FPMilan** [davestephens/ansible-nas#121]
(For config see #120)

Tested now on 3 different stations with the same result:
When enabling wallabag the site on port 7780 will load but without any style and without logo:

![image](https://user-images.githubusercontent.com/10432654/63685248-230aa980-c7ff-11e9-987f-066c17b43a8b.png)

Thanks for any help :)

**Comment by FPMilan** [comment]
More research has shown that this is a wallabag issue:
https://github.com/wallabag/docker/issues/115

**Comment by davestephens** [comment]
Sort of. 

We set the environment variable `SYMFONY__ENV__DOMAIN_NAME` to `https://wallabag.{{ ansible_nas_domain }}` as per https://github.com/wallabag/docker/issues/115.

I'll leave this open and do some testing per the fix in the linked issue (or you can :smile:)

**Comment by FPMilan** [comment]
That is correct for the case that one is using a domain name. If run locally this
`SYMFONY__ENV__DOMAIN_NAME="https://wallabag.{{ ansible_nas_domain }}"`
should be this instead:
`SYMFONY__ENV__DOMAIN_NAME="http://YOUR_IP:7780"`
 unfortunately I don't know how this would be done, steps would be though:

1. Implement a condition to activate the correct env variable based on the settings in `inventory` (localhost or DynDNS)
2. Have a function that discovers the local IP of the NAS or let the user set it somewhere.

**Comment by FPMilan** [comment]
Gave this another thought and had the following idea, don't know though if it will work but put it down for later testing:

add this to tasks/general.yml

```
    - shell: ip -o route get to 8.8.8.8 | sed -n 's/.*src \([0-9.]\+\).*/\1/p'
      register: my_ip
```
to set a variable with the name `my_ip' with the local ip address.

in the tasks/wallabag.yml

```
 env:
      SYMFONY__ENV__DOMAIN_NAME: "http://{{ my_ip }}:7780"

      SYMFONY__ENV__DOMAIN_NAME: "https://wallabag.{{ ansible_nas_domain }}"
      when: wallabag_available_externally == "true"
```

this should set env variable by default to something like 192.168.1.100:7780
but in the case that you set wallabag_available_externally to true is overwrites this to `https://wallabag.{{ ansible_nas_domain }}`

Will still need to test if it works

**Comment by bcurran3** [comment]
I'm hoping this gets fixed soon. I just discovered Wallabag due to ansible-NAS and like it - but right now it's FUGLY since it doesn't display correctly!

**Comment by bcurran3** [comment]
Whatever commit changed
```
    env:
      SYMFONY__ENV__DOMAIN_NAME: "http://{{ ansible_nas_domain }}"
```

to

```
    env:
      SYMFONY__ENV__DOMAIN_NAME: "https://wallabag.{{ ansible_nas_domain }}"
```

fixed the problem for me. (I manually changed it a few days ago.) It works and is BEAUTIFUL now!

Ironically it works via http://ip_address:7780 on my LAN now as well.
